apps/docs/apis/features/analytics.mdx

```diff
@@ -21,3 +21,61 @@ Our per key analytics give you a deep dive into each individual key, giving usag
 <Frame caption="Per key Analytics">
   <img src="/images/per-key-analytics.png" alt="Per key analytics"/>
 </Frame>
+
+
+## Tags
+
+You can add tags to verification requests to aggregate or filter data when querying.
+
+For example you might want to add the path of your API as a tag: `path=/v1/my-path` and then later retrieve a breakdown of api key usage per unique path.
+
+Unkey does not parse tags in any special way.
+The only limitations are:
+- Tags are strings.
+- You can specify up to 10 tags.
```

Comment:
Maybe, we should say 10 tags per request to make it a bit clear?

Reply:
Agreed